src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -148,6 +187,26 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
     }
   }, [approveCallback, handleCloseError, handleSetError, token?.symbol])
 
+  const handleRevokeApproval = useCallback(async () => {
+    // reset errors and close any modals
+    handleCloseError()
+
+    if (!revokeApprovalCallback) return
+
+    try {
+      // for pending state pre-BC
```

Comment:
BC?

Reply:
@alfetopito Any investor coming from before the year 0 ?

Reply:
Honestly I don't know what that means.
The closes BC means for me is British Columbia 🇨🇦